README.md

```diff
@@ -13,22 +13,19 @@
 
 ## Usage
 
-### Step 1. Install`passport-verify`
+The information here applies to services connecting using VSP 2.0.0 or above. If your connection involves a Matching Service Adapter
```

Comment:
The second sentence appears to be unfinished.  As Linguo would say - "sentence fragment".

Reply:
Oh I missed that one 🙈 Fixing now...